### ISSUE TITLE
Audit button - can we make it part of the prebid framework?

### ISSUE THREAD

**Issue by komplettsystem** [criteo/addressable-network-proposals#29]
https://github.com/criteo/addressable-network-proposals/blob/e131a358f40c4d4bb3e0ae863d2c3132fc6237bd/mvp-spec/dsp-api.md?plain=1#L547

@ischecht - maybe that is something you can provide some feedback on?
From my Product POV - this should be offered as part of the JS we provide as the prebid addressability framework, including some html representation of the audit log. A click listener on the ad itself shouldn't be too hard to register for this?

**Comment by jwrosewell** [comment]
Agree. Also we need to use some form of identification for the audit log so that browser extensions and web browsers can access the data.
